2-api/language.md

```diff
@@ -0,0 +1,44 @@
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Languages**
+
+# Languages served by Perspective API
+
+## Languages in production 
```

Comment:
My only hesitation with giving Language a separate doc is that we have to remember to update here and also in Model Reference when we have new launches. Maybe we could kill this page and rename Models to "Attributes and Languages" to better call out that language info is in there? Open to discussing!

Reply:
This was merged with the models page